hack/local-up-cluster.sh

```diff
@@ -769,20 +769,22 @@ function start_kubelet {
       fi
 
       docker run \
-        --volume=/:/rootfs:ro \
```

Comment:
I noticed that non-containerized kubelet uses a lot more flags: 

```
    	--vmodule="${LOG_SPEC}" \
        --container-runtime="${CONTAINER_RUNTIME}" \
        --feature-gates="${FEATURE_GATES}" \
        --cpu-cfs-quota=${CPU_CFS_QUOTA} \
        --cgroups-per-qos=${CGROUPS_PER_QOS} \
        --cgroup-driver=${CGROUP_DRIVER} \
        --keep-terminated-pod-volumes=${KEEP_TERMINATED_POD_VOLUMES} \
        --eviction-hard=${EVICTION_HARD} \
        --eviction-soft=${EVICTION_SOFT} \
        --eviction-pressure-transition-period=${EVICTION_PRESSURE_TRANSITION_PERIOD} \
        --fail-swap-on="${FAIL_SWAP_ON}" \
        ${auth_args} \
        ${dns_args} \
        ${cni_conf_dir_args} \
        ${cni_bin_dir_args} \
        ${net_plugin_args} \
        ${container_runtime_endpoint_args} \
        ${image_service_endpoint_args} \
	    ${KUBELET_FLAGS}
```

Some of them are probably not needed, but I needed `--cgroup-driver=${CGROUP_DRIVER}`.